message_passing/update_with_start/lazy_initialization/workflows.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+from decimal import Decimal
+from typing import List, Tuple
+
+from temporalio import workflow
+from temporalio.exceptions import ApplicationError
+
+from message_passing.update_with_start.lazy_initialization.activities import (
+    ShoppingCartItem,
+    get_price,
+)
```

Comment:
```suggestion
with workflow.unsafe.imports_passed_through():
    from message_passing.update_with_start.lazy_initialization.activities import (
        ShoppingCartItem,
        get_price,
    )
```

Need to encourage passing through all activity references and externally defined models instead of reloading them every workflow (for the samples this technically works because the activity doesn't do anything externally meaningful)

Reply:
I don't think I've ever once remembered to do this.